client/templates/topic/topicElement.js

```diff
@@ -45,6 +45,15 @@ Template.topicElement.onCreated(function () {
 });
 
 Template.topicElement.helpers({
+    getLabels: function() {
+        let aTopic = new Topic(this.minutesID, this.topic._id);
```

Comment:
This templates is also used in the MeetingSeries-Topics-View which displays all topics of all minutes of the selected series. In this case we do not have a `this.minutesID`. Instead we have to create a Topic-Object which lives inside of a meeting series. Therefore we have to pass the ID of the parent series as we did for example in line 104.

The current code leads to a client side exception on the Topics-Tab of a Meeting Series. Additionally the labels of the topics will not be displayed on this view.

To solve this issue replace the selected line with the following code:
```
let tmplData = Template.instance().data;
let parentElement = (tmplData.minutesID) ? tmplData.minutesID : tmplData.parentMeetingSeriesId;
let aTopic = new Topic(parentElement, this.topic._id);
```

Additionally, please change the access of `this.parentMeetingSeries` to `tmplData.parentMeetingSeriesId` which is a more cleaner way for accessing the templates data.